config/locales/es.yml

```diff
@@ -1448,110 +1449,110 @@ es:
           options:
-            12500_to_25000: "$12,500 - 25,000"
-            1_to_12500: "$1 - 12,500"
-            25000_to_40000: "$25,000 - 40,000"
-            40000_to_65000: "$40,000 - 65,000"
-            65000_to_73000: "$65,000 - 73,000"
+            12500_to_25000: De $12,500 a 25,000 dólares
+            1_to_12500: De $1 a 12,500 dólares
+            25000_to_40000: De $25,000 a 40,000 dólares
+            40000_to_65000: De $40,000 a 65,000 dólares
+            65000_to_73000: De $65,000 a 73,000 dólares
             over_73000: más de $73,000
             zero: "$0"
-        title: "¡Empecemos!"
+        title: "¡Vamos a empezar!"
     triage_income_types:
       edit:
         income_types:
-          farm: Gané dinero siendo dueño de una granja.
-          rental: Gané dinero de una propiedad de alquiler
-        income_types_label: "¿Alguna de las siguientes situaciones se aplica a usted?"
-        title: "¡Una última pregunta antes de que seleccionemos el servicio para usted!"
+          farm: Gané dinero por tener una granja
+          rental: Gané dinero de una propiedad alquilada
+        income_types_label: "¿Alguna situación a continuación le aplica a usted?"
+        title: "¡Una última pregunta antes de seleccionar un servicio para usted!"
     triage_referral:
       edit:
         help_text: |
-          Presente en línea por su cuenta utilizando un sitio web asociado certificado por el IRS. ¡Esta es nuestra opción más rápida para presentar sus impuestos y cobrar los créditos fiscales que se le deben!
+          Presente su declaración en línea por su cuenta utilizando el sitio web de un asociado. ¡Esta es nuestra opción más rápida para presentar sus impuestos y cobrar los créditos tributarios que se le deben!
 
-          Al hacer clic en continuar, será redirigido al sitio de nuestro socio para continuar con la presentación por su cuenta. El proceso es simple, rápido y fácil.
-        title: "¡Recomendamos realizar la presentación con nuestra opción gratuita File Myself!"
+          Al hacer clic en continuar, será dirigido al sitio web de nuestro asociado para que continúe presentando sus impuestos por su cuenta. El proceso es sencillo, rápido y fácil.
+        title: "¡Le recomendamos presentar su declaración con nuestra opción gratuita File Myself!"
     triage_start_ids:
       edit:
-        help_text: Puede ser útil reunir todos sus documentos de impuestos y la Tarjeta de Seguro Social o el papeleo de ITIN en este momento. Confirmaremos la información nuevamente más adelante en el proceso.
-        title: Comenzaremos preguntándole sobre sus documentos e identificaciones.
+        help_text: 'Le puede ser útil juntar todos sus documentos fiscales y su tarjeta de Número de Seguro Social o Número de Identificación Personal del Contribuyente (ITIN en inglés), en este momento. Confirmaremos la información nuevamente, más adelante en el proceso.  '
```

Comment:
There is an unnecessary ` ` (space) character at the end of this string. It is not a problem, but it is unnecessary.